src/styles/components/cards.module.css

```
@@ -0,0 +1,17 @@
+.card {
+  backdrop-filter: blur(2rem);
+  -webkit-backdrop-filter: blur(2rem);
+}
+
+.card__white {
+  background: rgba(255, 255, 255, 0.3) !important;
+  border: 1px solid rgba(0, 0, 0, 0.1) !important;
+}
+
+.card__white:hover {
+  border: 1px solid rgba(0, 0, 0, 0.3) !important;
+}
+
+.card__dark {
+  background: rgba(0, 0, 0, 0.3) !important;
+}
```

different design on dark mode? little bit confusing. And also the border can't be seen.
```suggestion
.card {
  backdrop-filter: blur(2rem);
  -webkit-backdrop-filter: blur(2rem);
}

.card__white {
  background: rgba(255, 255, 255, 0.2) !important;
  border: 1px solid rgba(0, 0, 0, 0.3) !important;
}

.card__white:hover {
  border: 1px solid rgba(0, 0, 0, 0.4) !important;
}

.card__dark {
  background: rgba(0, 0, 0, 0.2) !important;
  border: 1px solid rgba(255, 255, 255, 0.3) !important;
}

.card__dark:hover {
  border: 1px solid rgba(255, 255, 255, 0.4) !important;
}
```

---

Well, dark mode already have a really nice border, so there's no need to change it. As for the light mode, yes it does need some enhancement.

---

hmm yeh I kind of understand that but in my opinion the website should look similar to the OS itself and the default borders are not like the os design. Also for consistency reasons we should have the same design for both white and dark mode, shouldn't we?